runtime/doc/motion.txt

```diff
@@ -1083,6 +1083,95 @@ When you split a window, the jumplist will be copied to the new window.
 If you have included the ' item in the 'shada' option the jumplist will be
 stored in the ShaDa file and restored when starting Vim.
 
+							*jumplist-stack*
+
```

Comment:
there should not be an extra line here. (In general, check existing source to see conventions)

Reply:
Oops.  Fixed.